src/content/reference/cluster-upgrades/index.md

```diff
@@ -0,0 +1,167 @@
++++
+title = "Cluster Upgrades with Giant Swarm"
+description = "A detailed explanation how Kubernetes and other components are upgraded in a Giant Swarm installation"
+date = "2018-10-04"
+weight = 30
+type = "page"
++++
+
+# Cluster Upgrades with Giant Swarm
+
+A tenant cluster in a Giant Swarm installation is running a stack comprising many software components, provided by the Kubernetes project and other open source projects or software vendors, as well as by Giant Swarm.
+In order to keep all components up-to-date, to allow you to benefit from latest improvements, features and security fixes, we provide upgrades for the entire software stack in tenant clusters.
+
+At Giant Swarm we believe that frequent, small updates are a way to keep change in the system manageable.
+We invest a lot of effort in making this happen without disrupting your usage of your clusters.
+
+In this article, we explain how upgrades work in detail and how you should provide your workloads in order to keep them running during a tenant cluster upgrade and prevent disruptions of your applications.
+
+## Background and Concepts
+
+### The tenant cluster stack
+
+Among the third party components building a tenant cluster stack are
+
+- CoreOS Container Linux as the nodes' operating system
+- HashiCorp Vault as a public key infrastructure
+- Docker as a container runtime environment
+- CoreOS etcd as distributed storage for Kubernetes and Vault
+- Kubernetes with its many sub-components
+- Project Calico for virtual networking
+- CoreDNS for cluster-internal name resolution
+- NGINX Ingress Controller for connecting services with load balancers
+- Prometheus node exporter for hardware and OS metrics
+
+as well as many operators and controllers created and maintained by Giant Swarm.
```

Comment:
again I would not mention operators and outside components, this should be about how the user experiences upgrades, operator workings should move to architecture sections in the docs IMO

Reply:
Disagree. Totally fine to mention our own components here too.